sdks/python/apache_beam/runners/dataflow/internal/names.py

```diff
@@ -27,7 +27,13 @@
 # Standard file names used for staging files.
 from builtins import object
 
-DATAFLOW_SDK_TARBALL_FILE = 'dataflow_python_sdk.tar'
+# pylint: disable=unused-import
+from apache_beam.runners.internal.names import DATAFLOW_SDK_TARBALL_FILE
+from apache_beam.runners.internal.names import PICKLED_MAIN_SESSION_FILE
```

Comment:
I think we could avoid importing all of these except for PICKLED_MAIN_SESSION_FILE, and update all references in Beam codeabase to use apache_beam.runners.internal.names instead, similar what we do in apiclient.py.

```
# pylint: disable=unused-import
# Used by Dataflow legacy worker. 
from apache_beam.runners.internal.names import PICKLED_MAIN_SESSION_FILE
```

Reply:
@tvalentyn  Thank you for your help. Is it ok now?

Reply:
Please add the  comment `# Referenced by Dataflow legacy worker. ` before `from apache_beam.runners.internal.names import PICKLED_MAIN_SESSION_FILE`